src/Microsoft.DotNet.ProjectModel/LibraryDescription.cs

```diff
@@ -40,6 +42,7 @@ public class LibraryDescription
         public string Path { get; }
         public IEnumerable<LibraryRange> Dependencies { get; }
         public bool Compatible { get; }
+        public string PackagesDirectoryPath { get; }
```

Comment:
Spoke to @eerhardt offline about this. We decided to remove this property from `LibraryDescription`. It is already available on `PackageDescription` without any additions to that model. We can therefore `as` cast `LibraryDescription` in `DependencyContextBuilder` to try to get the path property for the `Library`.